### ISSUE TITLE
[I1-P4] - Índices

### ISSUE THREAD

**Issue by VicenteVicente** [PUC-IIC2283/2020-IIC2283-Pruebas#2]
Una duda muy corta, el return de ALG es:
 `(d0, ..., d2n-2)`
¿No debería ser?:
 `(d0, ..., dm-1)`

Saludos :)

**Comment by marceloarenassaavedra** [comment]
No, ALG tiene que retornar (d_0, ..., d_2n-2) como dice el enunciado.

Saludos!

Marcelo

**Comment by VicenteVicente** [comment]
Buena, ya me acordé del motivo gracias :)
